modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc

```diff
@@ -0,0 +1,69 @@
+[id='op-running-pipeline-and-task-run-pods-with-privileged-security-context']
+= Running pipeline run and task run pods with privileged security context
+:context: op-running-pipeline-and-task-run-pods-with-privileged-security-context
+
+toc::[]
+
+.Procedure
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
```

Comment:
with a `privileged`...

Reply:
Again, good catch. But your point made me think of this: In the context of `security context`, the word `privileged` is a proper noun, in the sense that it's a unique value with special meaning. As a correction, I used the article `the` before `privileged`.